freezeyt/url_finders.py

```diff
@@ -34,7 +37,9 @@ def get_html_links(
     base_url is the URL of the page.
     """
 
-    def get_links_from_node(node: xml.dom.minidom.Node, base_url) -> list:
+    def get_links_from_node(
+        node: xml.etree.ElementTree.Element, base_url: str
```

Comment:
Je to prkotina, ale zajímalo by mě proč i tohle nerozhodíš na dva řádky? Na první pohled to vypadá jako jeden argument s komplikovaným typem.

Reply:
klíč podle kterého zalamuji já je, že pokud se řádek nevleze do 79 znaků, tak se ho snažím smysluplně zalomit. většinou tak, že celý řádek odentruju o řádek níže, pokud se i potom nevleze na 79 znaků, tak jej většinou rozhodím, na každý řádek jeden parametr nebo také pokud je nějaký jeden hodně dlouhy parametr, tak jak je to  v tomto případě u všech ostatních. V tomto případě je to také dlouhý parametr - a proč jsem jej nechal takto? Popravdě nevím, možná, že jsem spěchal a odflákl nebo občas se mi stává, že jak programuji v každé volné chvíli a mám to programování hodně rozkouskováno tak na některé věci zapomenu nebo se pořádně nezamyslím.

Určitě mi nevadí, že mi opravuješ prkotiny rád bych psal kód tak, jak by se psát měl.